scipy/integrate/tests/test_quadrature.py

```diff
@@ -851,7 +851,7 @@ def simpson_reference(y, x):
 
         ref = cumulative_simpson(y, x=x)
         res = simpson_reference(y, x)
-        np.testing.assert_allclose(res[..., 1::2], ref[..., 1::2])
+        np.testing.assert_allclose(res[..., 1::2], ref[..., 1::2], 1e-15)
```

Comment:
Oops. I didn't mean to do this. Adjusting the tests will be the next step.
```suggestion
        np.testing.assert_allclose(res[..., 1::2], ref[..., 1::2])
```